### PR TITLE
Add support for -c / --cpu flag

### PR DESCRIPTION
**Pull request by Kurounin**
List configured cpu requests, limits and the current cpu usage when using -c / --cpu flag

### CODE REVIEW

File: kubecargoload.py [hunk 1]
```diff
@@ -43,13 +44,19 @@ def _fetch_pod_memory_usage(self):
...
             else:
-                name, _, memory_usage_pretty = columns
+                name, _, usage_pretty = columns
                 namespace = self._namespace
 
-            memory_usage = self._parse_quantity(memory_usage_pretty)
+            memory_usage = self._parse_quantity(usage_pretty)
 
             pod_key = (namespace, name)
             self._pod_memory_usage_data[pod_key] = memory_usage
```
Comment (eht16) [inline kubecargoload.py]: I would rewrite this part rather as this:
```py
            columns = line.strip().split()
            if len(columns) > 3:
                namespace, name, cpu_usage_pretty, memory_usage_pretty = columns
            else:
                name, cpu_usage_pretty, memory_usage_pretty = columns
                namespace = self._namespace

            usage_pretty = cpu_usage_pretty if self._cpu else memory_usage_pretty
            usage = self._parse_quantity(usage_pretty)

            pod_key = (namespace, name)
            self._pod_memory_usage_data[pod_key] = usage
```
This way it has less nested conditions and so is more readable, in my opinion.

File: kubecargoload.py [hunk 2]
```diff
@@ -363,6 +375,14 @@ def _setup_options():
         help='list the requested object(s) across all namespaces',
         default=False)
 
+    argument_parser.add_argument(
+        '-c',
+        '--cpu',
+        dest='cpu',
```
Comment (eht16) [inline kubecargoload.py]: Maybe `show_cpu_usage` is a more descriptive name for the destination option name?
If you agree, I would suggest to also use it for the corresponding attribute on `KubernetesCargoLoadOverviewProvider` and `KubernetesCargoLoadOverviewPrinter`.

The command line option name seems fine to me though.

**Review by eht16 — CHANGES_REQUESTED** [review]
Yay, thanks for the PR!
I was always too lazy to add CPU usage support as I didn't need it so far :).

It's great except two minor remarks I had.

File: kubecargoload.py (hunk 1 above)
Comment (Kurounin) [reply kubecargoload.py]: Sounds good. Will make the changes.

File: kubecargoload.py (hunk 2 above)
Comment (Kurounin) [reply kubecargoload.py]: Sounds good. Will make the changes.

File: kubecargoload.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 """
 List PODs of a specific namespace or all namespaces with their
-configured memory or cpu requests, limits and the current memory or cpu usage.
+configured memory requests, limits and the current memory usage.
```
Comment (eht16) [inline kubecargoload.py]: Did you remove the "cpu" part here on purpose?

Comment (Kurounin) [reply kubecargoload.py]: By mistake.

**Review by eht16 — APPROVED** [review]

**Comment by eht16** [comment]
Thanks for the PR!

**Comment by Kurounin** [comment]
Thanks for this amazing script! Glad I could contribute.

**Pull request merged**